src/state/sync/saga.js

```diff
@@ -254,7 +96,8 @@ function* initSync() {
     yield call(startSyncing)
   }
 
-  yield call(getSyncPref)
+  yield call(getSyncConf)
+  // yield call(getSyncPref)
```

Comment:
Why is this commented? 
Should be removed as we use syncConfigSaga instead right?

Reply:
Was renaming functions and leaving comments so nothing would be forgotten (forgot to delete this one), calling `getSyncConf` is correct